src/backend/d3d12/RenderPassDescriptorD3D12.cpp

```diff
@@ -57,7 +57,7 @@ namespace backend { namespace d3d12 {
     RenderPassDescriptor::GetSubpassOMSetRenderTargetArgs() {
         OMSetRenderTargetArgs args = {};
 
-        size_t rtvIndex = 0;
+        unsigned int rtvIndex = 0;
```

Comment:
This unrelated warning fix snuck in, but if that's ok I'll leave it in this commit.

Reply:
"Leave the campground cleaner than you found it" :)